### ISSUE TITLE
query re embedded host info in some MODS records

### ISSUE THREAD

**Issue by cwulfman** [PerseusDL/catalog_data#138]
See for example tlg0005.tlg001.opp-grc2.mods1.xml. Summarized, it is

_Idylls_, published in _Bucolicorum graecorum Theocriti, Bionis, Moschi reliquiae_ by Teubner in 1884 (2nd edition) as part of a series called _Bibliotheca scriptorum Graecorum et Romanorum Teubneriana. [Scriptores Graeci_. But _Bucolicorum graecorum_ also has a host element, _Poetae Graeci_.

What is the relationship between the series _Bibliotheca scriptorum..._ and _Poetae Graeci_?

(Aren't you glad I'm back??  :) )

**Comment by AlisonBabeu** [comment]
Hi @cwulfman, only you would assign a Github issue on a Friday night! :) So this record is kind of a weird one in that for once, I actually was cataloging a **physical book** that we had been given by the University of Chicago library.  They were de-accessioning a number of their Teubners and in this case, Poetae Graeci,  was the "binders title" (in that you only found it on the spine of the physical edition), given to a physical volume in which they had bound five separate Teubner editions that all involved Greek poetry.  

You can see this from the table of contents, 
```xml
  <mods:tableOfContents>Babrius. 1865.--Hesiodus. 1878.--Anacreon. 1876.--Theocritus.
1884.--Apollodorus. 1854.</mods:tableOfContents>
```
So there are actually a number of other MODS records that have this host information within them, since Poetae Graeci had five editions within it.  In sum, Bibliotheca Scriptorum is the Teubner series that is true for all five of these editions, and Poetae Graeci, is simply a "binders title" that was used for a physical manifestation that someone had created by binding together five Teubner editions of Greek poets.    In fact, this is actually reflected in the Worldcat record (http://www.worldcat.org/title/poetae-graeci/oclc/24188229), which if you note, only lists a University of Chicago edition. I didn't entirely know what to do with this volume back when I cataloged it sometime in 2008 or 2009, and I still don't now.

And yes, I'm glad you are back!

**Comment by cwulfman** [comment]
Sounds like the 5 editions are the crucial manifestations for the PC, right? And this _Poetae Graeci_ is some sort of re-publication--a one-off binding?  

Different from <https://archive.org/details/poetgraeci00eton>, which is a digital copy of a LoC copy of an edition of <http://www.worldcat.org/title/poetae-graeci-sive-selecta-ex-homeri-odyss-hesiodo-theocrito-cum-vulgata-versione-emendata-ac-variis-partim-scholiastarum-graecorum-partim-doctorum-recentiorum-notis-in-usum-regiae-scholae-etonensis/oclc/314794542/editions?editionsView=true&referer=br>, which we don't seem to have, right?

**Comment by AlisonBabeu** [comment]
Yes it is the 5 editions that are the crucial manifestations. this _Poetae Graeci_ is indeed a one off binding, as far as I can tell.  It is different from that Archive edition, which is not within the Perseus Catalog.
